proto/xudrpc.proto

```diff
@@ -455,16 +464,19 @@ message LndChannels {
   uint32 inactive = 2 [json_name = "inactive"];
   // The number of channels that are pending on-chain confirmation before they can be used.
   uint32 pending = 3 [json_name = "pending"];
+  // The number of channels that are pending on-chain confirmation before they can be used.
```

Comment:
This comment should say "The number of channels that have been closed" or something like that.